Auth0Tests/AuthenticationSpec.swift

```diff
@@ -808,6 +809,143 @@ class AuthenticationSpec: QuickSpec {
                     }
                 }
             }
+            
+            context("passwordless login") {
+                
+                let emailRealm = "email"
+                
+                it("should login with email code") {
+                    stub(condition: isToken(Domain) && hasAtLeast(["username": SupportAtAuth0, "otp": OTP, "realm": emailRealm, "grant_type": PasswordlessGrantType, "client_id": ClientId])) { _ in
+                        return authResponse(accessToken: AccessToken)
+                    }
+                    waitUntil(timeout: Timeout) { done in
+                        auth.login(email: SupportAtAuth0, code: OTP).start { result in
+                            expect(result).to(haveCredentials(AccessToken))
+                            done()
+                        }
+                    }
+                }
+                
+                it("should include audience if it is not nil") {
+                    stub(condition: isToken(Domain) && hasAtLeast(["audience": "audience"])) { _ in
```

Comment:
```suggestion
                    stub(condition: isToken(Domain) && hasAtLeast(["audience": "__test_audience__"])) { _ in
```

I've gotten into trouble with test data sharing a name with a key 😬

Reply:
Fixed in b718c8441f66d8259087aec2166fbac980cef0ce.